distro/README.md

```diff
@@ -32,7 +32,7 @@ riff-distro system download -m stable -o .
 Scan the release yaml files to generate a list of images into `image-manifest.yaml`.  In general this list will require additional validation. 
 
 ```sh
-riff-distro image list -m stable --no-check
+riff-distro image list -m manifest.yaml --no-check
```

Comment:
Don't you get the the same error on line 29? The `stable` manifest is no longer included with the `riff-distro`, only the `latest` is "built-in".